src/skins/vector/views/pages/Login.js

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
The components this pulls in are from the new src/components/ directory, split by functionality, but this new style non-split component is still part of the views hierarchy, mixed in with all the old, split components? 'views' were supposed to be the UI part of a component: this is a complete component, not a view?